apps-rendering/src/components/Deadline/index.tsx

```diff
@@ -3,7 +3,11 @@ import { isValidDate } from 'date';
 import type { FC } from 'react';
 import { highlight } from './styles';
 
-const Highlight: FC = ({ children }) => {
+type HighlightProps = {
+	children?: React.ReactNode;
```

Comment:
Is this optional?

Reply:
good querstion! I did a bit of digging on this pattern, and it looks like `@types/react` has a utility type [`PropsWithChildren`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f519573756dc171f38abdef9226862d16b487a62/types/react/ts5.0/index.d.ts#L778) for this. 

`PropsWithChildren` indicates that `children` is optional. I've updated this component to use `PropsWithChildren` instead, I think it's clearer.